eclair-core/src/test/scala/fr/acinq/eclair/blockchain/bitcoind/BitcoinCoreWalletSpec.scala

```diff
@@ -101,7 +101,7 @@ class BitcoinCoreWalletSpec extends TestKit(ActorSystem("test")) with BitcoindSe
     }
   }
 
-  test("handle error when signing transactions") {
+  test("handle errors when signing transactions") {
```

Comment:
wouldn't that be nice to have kept the single error message test just in case?

Reply:
It's not possible because errors are always returned in an array even if there is just one error.